packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -502,6 +507,41 @@ const TESTS: [string, any, Record<string, any>?][] = [
     ]],
   ])],
 
+  ['postLinkedAccountEmail', new Map<string, Test | any>([
+    ['subject', { test: 'equal', expected: 'New account linked to Firefox' }],
+    ['headers', new Map([
+      ['X-Link', { test: 'equal', expected: configUrl('initiatePasswordChangeUrl', 'account-linked', 'change-password', 'email') }],
+      // ['X-Linked-Account-Provider-Id', { test: 'equal', expected: PROVIDER['GOOGLE']}],
```

Comment:
This has been commented out until I understand what/how it should be coming through.